src/app/child-dev-project/schools/child-school-overview/child-school-overview.component.ts

```diff
@@ -33,6 +33,7 @@ import { PillComponent } from "../../../core/common-components/pill/pill.compone
   ],
   standalone: true,
 })
+// TODO: remove this and use PreviousRelationsComponent instead (once we have generic indices available for that)
```

Comment:
or should we do this right now and simply not use indices for the time being?

Reply:
I think we should do some very simple tests comparing requesting 1k documents with `loadType` and filtering them vs. having a prebuilt view and requesting the matching docs from there directly.

Reply:
**Performance tests:**
(on my fast machine with "CPU Throttling: 4x slowdown"; dev build ("npm start") on local machine; after initial indexing has completed; "session_type: local" with generated demo data, which has 4-5 ChildSchoolRelations per Child on average)
measurements are average of 3x reload on the ChildDetails tab + 3x opening from ChildrenList (which somehow was significantly slower)

| # Children   | ChildSchoolOverview.loadData (index) | RelatedEntitiesComponent.initData (no index) |
| --------- | ------------- | ------------- |
|   200    | 0.2 s  | 0.8 s  |
| 1000    | 1.0 s  | 13.9 s  |

PS.
... creating indices on first load for this size of demo data takes forever and a day :roll_eyes:

Reply:
Looks like the index does make a significant difference. I am following your suggestion and refactoring ChildSchoolOverview to also extend PreviousRelations now to remove unnecessary duplication.